lesson2/homework_4(prime_numbers).py

```diff
@@ -0,0 +1,12 @@
+import math
+if __name__ == '__main__':
```

Comment:
PEP8: two blank lines

Reply:
Like this? 
import math
blank
blank
if __name__ == '__main__':

Reply:
Done